### PR TITLE
[bitnami/jupyterhub] Updated README

### PR DESCRIPTION
**Pull request by vikram-bitnami**
**Description of the change**

Compressed and migrated information from the README

**Benefits**

Smaller and more compact README

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

This PR should not be merged until https://github.com/bitnami/charts-docs/pull/32 is landed and the corresponding links live on docs.bitnami.com

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/jupyterhub/README.md [hunk 1]
```diff
@@ -470,117 +415,67 @@ $ helm install my-release -f values.yaml bitnami/jupyterhub
...
+For more information, including how to provide a secret or a custom ConfigMap, refer to the [chart documentation on configuring Single User instances](https://docs.bitnami.com/kubernetes/infrastructure/jupyterhub/configuration/configure-single-user-instances/).
 
-### Restricting traffic using NetworkPolicies
+### Restrict traffic using NetworkPolicies
 
-The Bitnami JupyterHub chart enables NetworkPolicies by default. These restricts the communication between the three main components: the Proxy, the Hub and the Single User instances. There are two elements that were left open on purpose:
+The Bitnami JupyterHub chart enables NetworkPolicies by default. These restrict the communication between the three main components: the Proxy, the Hub and the Single User instances. There are two elements that were left open on purpose:
```
Comment (rafariossaa) [inline bitnami/jupyterhub/README.md:438]: Should it be:
`... This restrict...` instead of : `These restrict`?

Comment (vikram-bitnami) [reply bitnami/jupyterhub/README.md]: Hmm, I thought it was multiple policies, therefore made it plural. Fixed now

**Review by rafariossaa — APPROVED** [review]
LGTM

**Review by rafariossaa — APPROVED** [review]
LGTM

**Pull request merged**